### DISCUSSION TITLE
What is ngrok?

### DISCUSSION THREAD

**Discussion by SIXS1X** [adityash4rma/flaregram#10] [Q&A] [answered]
Can you explain what is ngrok and why do we use it in the development?

**Comment by adityash4rma** [comment, marked as answer]
Hey @SIXS1X, 
Ngrok is a software tool that allows you to expose a local server to the internet securely. We use it here for **Local Development and Testing**. By exposing local development servers and testing webhooks

You can read more about it here: https://ngrok.com/use-cases/ingress-for-dev-test-environments

PS: We aren't affiliated with ngrok in any way.
